doc/getting_started/getting_started.rst

```diff
@@ -34,7 +34,7 @@ You will need to have a `colcon <https://docs.ros.org/en/foxy/Tutorials/Colcon-T
   mkdir -p ~/ws_moveit2/src
   cd ~/ws_moveit2/src
 
-  git clone https://github.com/ros-planning/moveit2.git
+  git clone https://github.com/ros-planning/moveit2.git -b foxy
```

Comment:
Should we add a comment that we also support Rolling and Galactic, and ros2 branches should be used in that case?

Reply:
@tylerjw

Reply:
Yes, I think we could rethink the recommendations on this page entirely and encourage people using foxy to just use the debains of moveit.  If they want to develop on moveit they should probably be using rolling.

Reply:
I'll update the instructions here on website today (latest tomorrow)

Reply:
I'll close this PR then.

Reply:
@vatanaksoytezer I don't see the "-b foxy" change reflected in the website: http://moveit2_tutorials.picknik.ai/doc/getting_started/getting_started.html

Reply:
@lorepieri8 thanks for the heads up. PR is up to update this in #103. After https://github.com/ros-planning/moveit.ros.org/pull/623 gets merged everything will point moveit.ros.org which will contain the official build instructions. If you have any troubles with tutorials or source building or installation, feel free to open an issue and ping me. All the PRs should get merged pretty soon hopefully.